Makefile

```diff
@@ -8,34 +8,46 @@ LDFLAGS="-X $(PKG).version=$(VERSION) -X $(PKG).commit=$(COMMIT) -X $(PKG).date=
 .DEFAULT_GOAL := build
 
 .PHONY: all
-all: test cover fmt lint ci build generate
+all: test cover fmt lint ci build generate generated_up_to_date
```

Comment:
hmm i think this wouldn't need it, since it is part of the `ci`?

Reply:
Right, let me remove.